db/warehouse/migrate/20231120221840_colocated_warehouse_versions.rb

```diff
@@ -0,0 +1,17 @@
+class ColocatedWarehouseVersions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :versions do |t|
+      t.string   :item_type, null: false
+      t.bigint   :item_id,   null: false
+      t.string   :event,     null: false
+      t.string   :whodunnit
+      t.jsonb    :object
+      t.jsonb    :object_changes
+      t.datetime :created_at
+      t.string :session_id
+      t.string :request_id
+      t.references :user, index: false
```

Comment:
We could add an index but this table will probably be huge. Not sure if it's worth the cost.

Reply:
I don't think it's worth it if it makes insertion slower. its OK if the audit history views take a bit longer to load, that seems worth it.